main.go

```diff
@@ -154,6 +145,31 @@ func run(rs *Rsync, keys *drone.Key) error {
 	return session.Run(strings.Join(rs.Commands, "\n"))
 }
 
+// globSource returns the names of all files matching the source pattern.
+// If there are no matches or an error occurs, the original source string is
+// returned.
+//
+// If the source path is not absolute the root path will be prepended to the
+// source path prior to matching.
+func (rs *Rsync) globSource(root string) []string {
```

Comment:
It might make sense to add this functionality to the drone API, i.e.

```go
func (w *drone.Workspace) Glob(pattern string) ([]string, error)
```

Ditto for `writeKey`:

```go
func (w *drone.Workspace) WriteKey() error
```